### ISSUE TITLE
v4 typings

### ISSUE THREAD

**Issue by twawszczak** [luin/ioredis#669]
Is there a plan to include typings for lib inside build, or it still has to be in @types repo?

**Comment by luin** [comment]
Yes, ioredis will end up including typings in the module when all the code have been migrated to TypeScript.

Get Outlook<https://aka.ms/qtex0l> for iOS
________________________________

**Comment by egoarka** [comment]
@luin when are you planning to migrate code to TypeScript? I want to participate in the development

**Comment by luin** [comment]
@egoarka The migration has already started. Pull requests that migrating JS to TS are welcome!
